lib/cluster-store.js

```diff
@@ -59,6 +59,7 @@ module.exports = function(connectOrSession) {
   function ClusterStore(options) {
     Store.call(this, options);
     this._collection = NativeStore.collection(COLLECTION_NAME);
+    this._collection.configure(options);
```

Comment:
In documentations says that the options object is to initialise ClusterStore
https://apidocs.strongloop.com/strong-cluster-connect-store/v/1.0.0/
@param {Object} options Options for the ClusterStore object.
So I sought that it should to passed to configure method.
So, I would say it's better to move collection config to options.config because method config should be called after setting collection name.